Services/UserService/Utils/TokenMiddleware.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using UserService.Properties;
+
+namespace UserService.Utils
+{
+    public class TokenMiddleware
+    {
+        private RequestDelegate next;
+
+        public TokenMiddleware(RequestDelegate next)
+        {
+            this.next = next;
+        }
+        
+
+        public async Task InvokeAsync(HttpContext context)
+        {
+            if (context.Request.Method == "POST" 
+                & context.Request.Path == StringConstants.UrlCreateUser)
```

Comment:
Please read about difference between & and &&. in this case we need && and you will tell me why on next session